package.json

```diff
@@ -4,21 +4,15 @@
   "description": "",
   "author": "Front Spot",
   "scripts": {
-    "prestart": "babel-node tools/startMessage.js",
-    "start": "npm-run-all --parallel test start:backend lint:watch",
-    "start:backend": "cross-env NODE_ENV=development nodemon app --exec babel-node",
-    "open:src": "cross-env NODE_ENV=development babel-node app",
+    "prestart": "npm run docs",
+    "start": "npm-run-all --parallel test start:server lint:watch ",
+    "start:server": "cross-env NODE_ENV=development babel-node app",
```

Comment:
With `nodemon` server is restarting every time when files are changed. Is it the same behavior now?